build/root/WORKSPACE

```diff
@@ -66,3 +50,10 @@ docker_pull(
     registry = "gcr.io",
     repository = "google-containers/debian-iptables-amd64",
 )
+
+docker_pull(
+    name = "official_busybox",
+    registry = "index.docker.io",
+    repository = "library/busybox",
+    tag = "latest",  # TODO: more hermetic?
```

Comment:
one issue with using a tag (either `latest` or a minor version, e.g. `1.26`) is that I'm not sure when bazel would pull in a new version. I don't think we can force pulling like can with `docker build`, but I need to confirm.